tests/test_util.py

```diff
@@ -415,42 +411,15 @@ def request_callback(request, all_headers=True):
              {
                  'url': "https://some_registry.io",
                  'insecure': False,
-                 'auth': {
-                     'cfg_path': '/not/important'
-                 }
              }
-         ]
+         ],
+         'registries_cfg_path': '/not/important',
      },
      {
          'uri': "https://some_registry.io",
          'insecure': False,
          'dockercfg_path': '/not/important'
      }),
-    # Have both source_registry and pull_registries, matches pull_registries
-    ('some_registry.io',
```

Comment:
Why is this case not needed anymore?

Reply:
Good catch. Fixed both.

I think the reason should be I didn't have a good understand of `RegistrySession.create_from_config`.